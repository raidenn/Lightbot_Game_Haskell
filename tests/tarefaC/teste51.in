A
A
0 0 N
LLLLLAL
