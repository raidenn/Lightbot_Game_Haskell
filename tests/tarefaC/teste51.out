0 0
0 0
0 0
0 0
0 0
0 1
FIM 7
